packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -31,16 +31,28 @@ import { store as blockEditorStore } from '../../store';
 
 const VOICE_OVER_ANNOUNCEMENT_DELAY = 1000;
 
-const defaultRenderToggle = ( { onToggle, disabled, style, onLongPress } ) => (
+const defaultRenderToggle = ( {
+	canViewEditorOnboarding,
+	onToggle,
+	disabled,
+	style,
+	onLongPress,
+} ) => (
 	<ToolbarButton
-		title={ __( 'Add block' ) }
+		title={
```

Comment:
👋 @dcalhoun ! I see that the `title` prop is being reused(?) for hosting the tooltip content. Can you elaborate on this in the PR description, to add some info on what's going on here, how's the tooltip functionality implemented for native emobile for instance. A link to some resource that might already have the info would do it too. Does that make sense?

Reply:
> 👋 @dcalhoun ! I see that the `title` prop is being reused(?) for hosting the tooltip content. Can you elaborate on this in the PR description, to add some info on what's going on here, how's the tooltip functionality implemented for native emobile for instance. A link to some resource that might already have the info would do it too. Does that make sense?

Definitely. I have updated the PR description with additional details as to _why_ these props are used, which I will copy here for convenience.

>The `Button` component [already renders the existing Tooltip component](https://github.com/WordPress/gutenberg/blob/b955783c7b76a4518dbc47f867a4ff3cdf03f1d6/packages/components/src/button/index.native.js#L188-L194). Because of this, it was not necessary to wrap the block inserter dropdown menu button in a new `Tooltip` element. Instead, this PR leverages `Button`'s `title` and `showTooltip ` props to display new "Tap to add content" tooltip required for the onboarding feature. One benefit of this approach is that we do not need to worry about forwarding a React ref through to `Button`. 

We could, of course, avoid this approach and merely wrap the existing `ToolbarButton` with `Tooltip` if that is preferred. We would just need to make sure the ref is forwarded to the `Button`, which currently is not set up. I reverted that support in https://github.com/WordPress/gutenberg/pull/32001/commits/ff738326f7edcec65686f4957f6376eb0293bc18#diff-4faa11aaadbb56229d147af80dfa880a68aa32b647c60c7690e710ef1d7db1d8.

Let me know what you think or if I failed to explain anything clearly. Thanks!

Reply:
Thanks for elaborating!

> We could, of course, avoid this approach and merely wrap the existing ToolbarButton with Tooltip if that is preferred.

FWIW, I'm cool with the current approach to reuse Button's title/showTooltip API. I was only interested in surfacing/documenting the implementation detail. 👍